corporate/lib/stripe.py

```diff
@@ -787,6 +787,35 @@ def get_past_invoices_session_url(self) -> str:
             return_url=f"{self.billing_session_url}/billing/",
         ).url
 
+    def get_stripe_customer_portal_url(
+        self, manual_license_management: bool, tier: Optional[int] = None
+    ) -> str:
+        customer = self.get_customer()
+        assert customer is not None and customer.stripe_customer_id is not None
+
+        if tier is None:
+            return_url = f"{self.billing_session_url}/billing/"
+        else:
```

Comment:
might be better to just pass a parameter which tells us if this is billing page rather than doing an indirect check which might not be true in future.

Reply:
Makes sense. Added a `return_to_billing_page` parameter that's set to true for the billing page.